File: BackEndAPI/Data/Entities/HinhAnh.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace BackEndAPI.Entities
{
    public partial class HinhAnh
    {
        public int MaHinhAnh { get; set; }
        public int MaSp { get; set; }
        public string NguonHinhAnh { get; set; }
        public bool? MacDinh { get; set; }

        public virtual SanPham SanPham { get; set; }
    }
}

File: BackEndAPI.Java/DiChoThue/src/main/java/com/example/DiChoThue/Entities/HinhAnh.java
package com.example.DiChoThue.Entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "HINH_ANH")
public class HinhAnh {
    @Id
    private int MA_HINH_ANH;
    private int MA_SP;
    private String NGUON_HINH_ANH;
    private Boolean MAC_DINH;
}
